### ISSUE TITLE
BROKEN HITBOX

### ISSUE THREAD

**Issue by SC3-Gazzy-Gaming** [FredyH/Perpheads-Issues#625]
EXCUSE ME,SHOT ALFA IN THE HEAD,SURVIVED

**Comment by LaCreepis** [comment]
Is this a different situation or are you referencing this sit that I uploaded? https://gfycat.com/AgedInformalJaguarundi

**Comment by SC3-Gazzy-Gaming** [comment]
Yes but also how with myself and Edward field where we shot him alot in the head and sponged them all.
